src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -74,16 +73,15 @@ const ActiveState = ({
 
     return (
         <ul className="bcs-activity-feed-active-state">
-            {items.map((item: any) => {
-                const { type, id, versions, permissions } = item;
+            {items.map((item: FeedItem) => {
                 const isFocused = item === activeEntry;
                 const refValue = isFocused ? activeFeedItemRef : undefined;
 
-                switch (type) {
+                switch (item.type) {
```

Comment:
destructuring changes the type from an enum of literals to an enum including `string`, which doesn't allow the case statements to be refined for each item type